test/policies/account_policy_test.rb

```diff
@@ -65,9 +64,9 @@ class AccountPolicyTest < PolicyTest
     test 'non teammember cannot update account' do
       refute_permit alice, account2, :update?
     end
-    
+
     test 'for team enabled api user cannot update account' do
```

Comment:
a team enabled api user should be able to update this account, shouldn't he ?
probably just the wrong user ?!

please think about all possible use cases ! 
write them somewhere down and check if you got all possible cases in here !